src/utils/self_test/SConscript

```diff
@@ -9,6 +9,18 @@ import os
 import daos_build
 
 SELF_TEST = 'self_test.c'
+LIB_CONFIG_INI = 'src/configini.c'
+LIB_CONFIG_CNF = 'etc/selftest.cnf'
+LIB_CONFIG_ECNF = 'etc/selftest_expected.cnf'
+
+# Function to filter out gcc flags from this compilation env
+def remove_certain_gcc_flags(str):
```

Comment:
(pylint-redefined-builtin) Redefining built-in 'str'
(pylint-missing-docstring) Missing function docstring